dmoj/executors/GO.py

```diff
@@ -5,6 +5,7 @@
 
 reinline_comment = re.compile(br'//.*?(?=[\r\n])')
 recomment = re.compile(br'/\*.*?\*/', re.DOTALL)
+repackage = re.compile(br'\s*package\s+main\s*;?\b')
```

Comment:
Why do you have a `\b` in the end? It's pointless, for if the character that follows is `\w`, it will always match by not matching `;`. If the character that follows is not `\w`, it can always match by making both `\s` and `;` not match anything and use `\b` as boundary between `main` and that character.

So essentially, you have `\s*package\s+main\b`.